superset-frontend/src/components/Loading/index.tsx

```diff
@@ -20,6 +20,8 @@
 import React from 'react';
 import { styled } from '@superset-ui/core';
 import cls from 'classnames';
+// @ts-ignore
```

Comment:
what ts issue is this ignoring?

Reply:
@arash TypeScript does not like the image import, bit this is valid for webpack to import the resource